### DISCUSSION TITLE
Existing Pull Requests

### DISCUSSION THREAD

**Discussion by WalterCouto** [antlr/antlr4#4694] [General]
There are a few existing Pull Requests reviewed that seem like good updates across several platforms that seem to be ignored over the last few releases. Are there any plans to actually go through and clean up the PRs and include them in a release or reject them?

**Comment by jimidle** [comment]
I intend to do that for the Go target, but we just had a release, so such that are accepted will be in the next release. I cannot speak to other targets though.
